client/verta/verta/endpoint/resources.py

```diff
@@ -39,30 +39,30 @@ class Resources(object):
     .. code-block:: python
 
         from verta.endpoint.resources import Resources
-        resources = Resources(cpu_millis=250, memory="512Mi")
+        resources = Resources(cpu=.25, memory="512Mi")
 
     """
-    CPU_ERR_MSG = "`cpu_millis` must be int greater than 0"
+    CPU_ERR_MSG = "`cpu` must be a number greater than 0"
     MEMORY_ERR_MSG = ' '.join([
         "`memory` must be a string representing a plain integer",
         "or a fixed-point integer with suffixes",
         "E, P, T, G, M, K, Ei, Pi, Ti, Gi, Mi, Ki;",
         "for example: 128974848, 129e6, 129M, 123Mi",
     ])
 
-    def __init__(self, cpu_millis=None, memory=None):
-        if cpu_millis is not None:
-            self._validate_cpu(cpu_millis)
+    def __init__(self, cpu=None, memory=None):
+        if cpu is not None:
+            self._validate_cpu(cpu)
```

Comment:
@conradoverta Do you think I should warn the user if they're passing in an unreasonably high number for `cpu` (e.g. `1500`, if they've started using this when it was `cpu_millis`).

I could _also_ check for `cpu_millis` as a keyword argument.

Reply:
We haven't exposed this yet
I generally prefer leaving validation to the backend